lib/resque/pool.rb

```diff
@@ -368,6 +416,7 @@ def create_worker(queues)
       worker = ::Resque::Worker.new(*queues)
       worker.verbose = ENV['LOGGING'] || ENV['VERBOSE']
       worker.very_verbose = ENV['VVERBOSE']
+      worker.fork_per_job = fork_enabled_for_queues?(queues)
```

Comment:
The `fork_per_job` attribute is only available since resque 1.23.1. We should take this into account and warn that resque is too old if the attribute does not exist, or bump the resque dependency in the gemspec.

Reply:
Yes! You are right! I updated the resque versions constraints because IMHO this is the right thing to do! You can check e5000501c585a57179228958d5e5e3896526552e for more info.